sdk/monitor/opentelemetry-exporter-azure-monitor/api-extractor.json

```diff
@@ -9,9 +9,9 @@
     "reportFolder": "./review"
   },
   "dtsRollup": {
-    "enabled": false,
```

Comment:
Calling out changed in this file - wasn't sure the impact so I'd like to have someone 👍 or 👎 this

Reply:
@xirzec do you happen to know why this was disabled?

Reply:
Api-extractor support came in very late/hot before the archboard review, so I think it was simply that I didn't have time to test switching over package.json to reference the rollup (right now it's `"types": "types/src/index.d.ts",`)

I'm fine with this change, but we should update package.json as well to reference this new file.